### PR TITLE
Configure Root Logger for logging

### PR DESCRIPTION
**Pull request by shaardie**
Instead of creating a new logger for all our modules, we can simply change the
configuration of the root logger instead.

This does have the nice affect that every imported module also use this
configuration and therefore have the same formatting and log level.

Downside is that if the module is not executed via its generated binary but as
a module (`python -m emissionsapi.*`), the logged name will be `__main__`. If the
module is executed via its generated binary instead, the name matches the
module, e.g. `emissionsapi.download`.

Signed-off-by: Sven Haardiek <sven@haardiek.de>

### CODE REVIEW

File: emissionsapi/db.py [hunk 1]
```diff
@@ -6,6 +6,7 @@
 """
 
 from functools import wraps
+import logging
```
Comment (TylerBird) [inline emissionsapi/db.py]: Is "logging" an external package that you found?

Comment (lkiesow) [reply emissionsapi/db.py]: `logging` is [part of the standard library of Python](https://docs.python.org/3.7/library/logging.html)

File: emissionsapi/db.py [hunk 2]
```diff
@@ -14,10 +15,9 @@
 import geoalchemy2
 
 from emissionsapi.config import config
-import emissionsapi.logger
 
 # Logger
-logger = emissionsapi.logger.getLogger('emission-api.db')
+logger = logging.getLogger(__name__)
```
Comment (TylerBird) [inline emissionsapi/db.py]: Where does "__name__" come from? Is it a define?

Comment (TylerBird) [reply emissionsapi/db.py]: Ahh I tried to type `__name__` now I got it :D

**Pull request merged**